crypto/codec/amino.go

```diff
@@ -29,9 +28,6 @@ func RegisterCrypto(cdc *codec.LegacyAmino) {
 	cdc.RegisterInterface((*cryptotypes.PubKey)(nil), nil)
 	cdc.RegisterConcrete(sr25519.PubKey{},
 		sr25519.PubKeyName, nil)
-	// TODO Same as above, for ED25519
-	cdc.RegisterConcrete(tmed25519.PubKey{},
```

Comment:
Now we use sdk keys in the sdk and make them tm proto keys when we send them to Tendermint.